app/src/main/java/net/squanchy/favorites/view/FavoritesViewHolders.kt

```diff
@@ -27,6 +28,10 @@ class EventViewHolder(itemView: EventItemView) : FavoritesViewHolder<EventItemVi
 class HeaderViewHolder(itemView: TextView) : FavoritesViewHolder<TextView>(itemView) {
 
     fun updateWith(date: LocalDate) {
-        (itemView as TextView).text = date.toString("EEEE d")
+        (itemView as TextView).text = date.format(formatter)
+    }
+
+    companion object {
+        val formatter = DateTimeFormatter.ofPattern("EEEE d")
```

Comment:
⚠️ I would not put this in the companion object; in fact, this is potentially source of errors if you statically cache the formatter in the companion object and the user changes the locale of the device. This should be a (private!) property instead

Reply:
How about with with @quiro91's suggestion and provide them as qualified bindings?

Reply:
Would be 💯